index.html

```diff
@@ -2,179 +2,219 @@
+            <p>Hello I’m a software developer! I can help you build a
+                product , feature or website Look through some of my
+                work and experience! If you like what you see and have
+                a project you need coded, don’t hesitate to contact.<br><br> <span>LETS CONNECT</span></p>
+                <ul class="icons">
+                    <li><a href="#" target="_blank" aria-label="Github"><i class="fab fa-github"></i></a></li>
+                    <li><a href="#" target="_blank" aria-label="linkedin"><i class="fab fa-linkedin-in"></i></a></li>
+                    <li><a href="#" target="_blank"><i class="fab fa-angellist"></i></a></li>
+                    <li><a href="#" target="_blank" aria-label="twitter"><i class="fab fa-twitter"></i></a></li>
+                    <li><a href="#" target="_blank" aria-label="Medium"><i class="fa-solid fa-m"></i></a></li>
 
-                
-             
-               
-            </section>
-    </section>
-     
-    
+                </ul>
+                <button class="about_btn">Get My Resume</button>
+            </div>
+            <div class="languages">
+                <div class="lan_arrow">
+                <h5>Languages</h5>
+                <i class="fas fa-angle-down"></i>
+                </div>
+                <div class="img">
+                <ul class="lan">
+                    <li class="html"><img src="images/Frame 9.svg" alt=" html logo"></li>
+                    <li class="css"><img src="images/Frame 10.svg" alt="css logo"></li>
+                    <li class="java"><img src="images/Frame 11.svg" alt="java logo"></li>
+                </ul>
+            </div>
+                <div class="about_work">
+                    <h6>Frameworks</h6>
+                     <i class="fas fa-angle-right"></i>
+                </div>
+                <div class="about_skill">
+                    <h6>Skill</h6>
+                    <i class="fas fa-angle-right"></i>
+                </div>
+            </div>
+        </section>
+        <section class="contact">
+            <div class="contact_me">
+                <h6>Contact me</h6>
+                <p>If you have an application you are interested in
+                    developing, a feature that you need built or a projectthat needs coding. I’d love to help with it!</p>
+                    <form action="https://formspree.io/f/mjvdadbe" method="post" class="inputs">
+                    <input type="text" name="Full name" placeholder="Lorem Ipsum" maxlength="30" required class="input1">
+                    <input type="text" name="Email" placeholder="loremipsum@gmail.com"  required class="input1">
+                    <textarea name="messege" id="message" cols="30" rows="5" placeholder="Write your message here..." required maxlength="500" class="text"></textarea>
+                    <button class="contact_btn" type="submit">Get in touch</button>
+                </form>
```

Comment:
- [ ] Nice work so far 👍🏿 on your project, on inspection I, not i little difference in your contact form border-radius compared to the Figma design. I kindly suggest you add the `border radius` to your contact form as per the project requirement